src/gcpsample_client/resolve.py

```diff
@@ -21,10 +21,6 @@
 # We describe what we want to do with the asset
 context.access = context.access.kRead
 
-# We describe the lifetime of the returned reference
-# as persistent retention may require a more stable value.
-context.retention = context.retention.kTransient
-
```

Comment:
https://github.com/OpenAssetIO/OpenAssetIO/issues/1048

Reply:
Ok, I'll remove this. And no one will care that it's not in the context object, yeah?